Management/src/main/java/com/stack/jspdemo/controller/WelcomeController.java

```diff
@@ -0,0 +1,15 @@
+package com.stack.jspdemo.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+public class WelcomeController {
```

Comment:
Is it needed to use public here?

Reply:
Done but I have a question, is this because I do not need access this class outside the package?